### DISCUSSION TITLE
两次Ctrl+c无效。

### DISCUSSION THREAD

**Discussion by earthmoon** [xiaoyifang/goldendict-ng#793] [Q&A]
您好，

很多时候在Acrobat软件中，按两次Ctrl+c无效，有时又可以，6+版本存在这个问题，5开头的版本没有这个问题。只要把goldendict主界面调出来，再按两次ctrl+c就可以使用，不调出来基本不行。我感觉很可能和电脑有关，该怎么找到问题症结，彻底修复这个问题？

感觉问题处在这，有时按两次ctrl+c在屏幕左上角出现这个：
![image](https://github.com/xiaoyifang/goldendict-ng/assets/45897471/a9fc8775-09a2-429e-a8d5-c5eb686bc810)

谢谢！

**Comment by xiaoyifang** [comment]
右键以 管理员  运行  goldendict    ，试一下

**Reply by earthmoon** [reply]
> 右键以 管理员 运行 goldendict ，试一下

谢谢。还是一样哎。
